frontend/src/metabase/admin/datamodel/components/GuiQueryEditor/GuiQueryEditor.module.css

```diff
@@ -29,5 +29,5 @@
 /* create a smaller inset arrow on the right */
 .arrowRight::after {
   right: -19px;
-  border-left-color: var(--color-white);
+  border-left-color: var(--mb-color-bg-white);
```

Comment:
Just to close the loop. I think it makes sense that the border follows the background color since they both are connected. And it's been used throughout the PR.